wpeverest/user-registration
# Tweak - WPML compatibility for static strings

### All Submissions:




### Changes proposed in this Pull Request:
This PR covers all static strings which need to be translated in the frontend via WPML Plugin.



### How to test the changes in this Pull Request:

1. Visit site and check all screnaio like validation, success message, profile menu, dashboard content, password strenght button name, change password content, reset password content.
2. All above content strings must show in string translation list in user-registration domain.
3. Try to translate in different languages and check whether it's working or not.

### Types of changes:




### Other information:




### Changelog entry

> Tweak - WPML compatibility for static strings
